odra-casper/test-vm/src/vm/casper_vm.rs

```diff
@@ -50,17 +70,18 @@ pub struct CasperVm {
 impl CasperVm {
     /// Creates a new instance with predefined accounts.
     pub fn new() -> Rc<RefCell<Self>> {
-        Rc::new(RefCell::new(Self::new_instance()))
+        let vm = Self::new_instance();
```

Comment:
Showing Ola how PR process works.
Why did you do that? It was so nice... 😿

Reply:
Fixed in f18044d